README.md

```diff
@@ -78,56 +80,93 @@ DATABASES = {
             'local_infile': 1,
         }
     }
-```
-Change the password field to the password you chose when you installed MySQL. 
-Alterntively, you can copy `settings.py` to `settings_local.py` and modify that local version of it.
-django will use `settings_local.py` if it's there.
+  }
+  ```
+
+  Change the password field to the password you chose when you installed MySQL. 
+
+  Alternatively, you can create `disclosure_backend/project/settings_local.py` and add this info into it.
+  Django will override values in `settings` with those from `settings_local.py`, if `settings_local.py` exists.
 
+3. Run the database migration scripts
+  ```
+  python manage.py migrate
+  ```
+  
+  OSX: If you get the following error `django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module: dlopen(_mysql.so, 2): Library not loaded: libssl.1.0.0.dylib`
+  
+  Then, you need to add openssl to your `DYLD_LIBRARY_PATH`:
+  1. Go to `/usr/local/Cellar/openssl/`, and locate your directory (e.g. 1.0.2d_1)
+  2. Add the following to your `~/.bash_profile`:
+   ```
+   export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:/usr/local/Cellar/openssl/1.0.2d_1/lib
+   ```
```

Comment:
Also, I haven't had to do this on the couple Mac OS laptops that I've installed python projects on, hmmmm.